main/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,77 @@
+import AccordionItem, {AccordionData} from './AccordionItem';
+import {Animated, StyleProp, ViewStyle} from 'react-native';
+import React, {FC, ReactElement, useRef} from 'react';
+
+import styled from '@emotion/native';
+
+const Container = styled.View`
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const TOGGLE_ELEMENT_PROPERTY = 'toggleElement';
+export interface Styles {
+  titleContainer?: StyleProp<ViewStyle>;
+  bodyContainer?: StyleProp<ViewStyle>;
+}
+
+export interface AccordionBaseProps<T> {
+  data: T;
+  style?: ViewStyle;
+  styles?: Styles;
+  shouldAnimate?: boolean;
+  collapseOnStart?: boolean;
+  animDuration?: number;
+  activeOpacity?: number;
+  [TOGGLE_ELEMENT_PROPERTY]?: ReactElement | null;
+  renderTitle?: (item: string) => ReactElement;
+  renderBody?: (item: string) => ReactElement;
```

Comment:
I think `render` functions should not be optional.

Reply:
I think ```render``` functions are exposed to user for scalabiilty separately from default implemented design. 
if it not be optional property, ```render``` functions will always have to be implemented.
it's on [contributing guide](https://dooboo-ui.dooboolab.com/?path=/docs/contributing-contributing--page#coding-guidelines) and tell me more about what I missed or what you think.
thanks review!